### PR TITLE
Update RFD 72 (EC2 tags) with Azure info

### PR DESCRIPTION
**Pull request by atburke**
This PR updates RFD 72 to include information regarding Azure tags.

### CODE REVIEW

File: rfd/0072-ec2-tags.md [hunk 1]
```diff
@@ -26,21 +29,36 @@ As of January 2022, [instance tags are available via the instance metadata servi
...
-### Throttling
+Namespace prefix: `aws`
+
+#### Throttling
+
 AWS applies [per-instance throttling to instance metadata requests](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html#instancedata-throttling). Each instance can have at most 50 tags, so with one request to fetch all tag keys and up to 50 requests to fetch tag values, Teleport only needs to make at most 51 instance metadata requests per hour, which should not have throttling issues.
 
+### Azure
+
+Azure instance metadata (including tags) is always enabled and requires no additional setup from the user. Azure VMs do not require a restart for tag updates to apply.
```
Comment (jakule) [inline rfd/0072-ec2-tags.md:52]: Always enabled means that it is enabled and cannot be disabled, or is it enabled by default, and there is a way to disable it?
I'd assume the first, but I want to double check.

Comment (jakule) [inline rfd/0072-ec2-tags.md:52]: Are there any limits on how many tags we can fetch on one request or overall limits on tags?

Comment (atburke) [reply rfd/0072-ec2-tags.md:52]: It can't be disabled.

Comment (atburke) [reply rfd/0072-ec2-tags.md:52]: No limits on how many tags we can fetch at once. Azure instances can have at most 50 tags.

Comment (jakule) [reply rfd/0072-ec2-tags.md:52]: @atburke Please add that 50 tags limit to RFD.

**Review by jakule — APPROVED** [review]

**Review by Joerger — APPROVED** [review]

**Pull request merged**